Assets/MixedRealityToolkit/Definitions/MixedRealitySpatialObserverConfiguration.cs

```diff
@@ -39,6 +39,9 @@ public struct MixedRealitySpatialObserverConfiguration : IMixedRealityServiceCon
         [SerializeField]
         private BaseSpatialAwarenessObserverProfile observerProfile;
 
+        /// <inheritdoc />
+        public BaseMixedRealityProfile Profile => observerProfile;
+
         /// <summary>
         /// Spatial Observer specific configuration profile.
```

Comment:
Should this be marked `Obsolete`, like you did in MixedRealityServiceConfiguration?

Reply:
No because they all cast to the appropriate profile type. InputSystem is just weird that it does not have a unique BaseInputConfiguration class. ServiceConfiguration is meant to be generic and hence why I obsolete that one